pkg/gits/git_cli.go

```diff
@@ -869,6 +869,11 @@ func (g *GitCLI) FetchTags(dir string) error {
 	return g.gitCmd(dir, "fetch", "--tags")
 }
 
+// FetchRemoteTags fetches all the tags from a remote repository
+func (g *GitCLI) FetchRemoteTags(dir string, repo string) error {
```

Comment:
Might be better to extend the command above, and check if repo is empty?

Reply:
Yeah, I think I'm gonna do a dive through `git_cli` at some point soon and clean up a bunch of stuff.